pkg/controller/actions/render/kustomize/action_render_manifests_test.go

```diff
@@ -206,7 +206,7 @@ func TestRenderResourcesWithCacheAction(t *testing.T) {
 
 	render.RenderedResourcesTotal.Reset()
 
-	for i := int64(0); i < 3; i++ {
+	for i := range 3 {
```

Comment:
not sure why the old code set to use int64 not just int, otherwise i am fine to use range 

/LGTM

Reply:
@lburgazzoli it looks like you were the one to write this originally; can you remember if there's any reason for the cast to int64 here?

Reply:
I think it is a leftover of one of the various iteration to get at this point, since I don't see a reason to have it explicitly set to `int64` with the current code. If my memory assists me, It was because I had to cast the index to a `int64` a number of time so I oped to do it once at declaration time.